streams/readable-streams/brand-checks.js

```diff
@@ -148,4 +148,26 @@ test(() => {
 
 }, 'ReadableStreamController.prototype.error enforces a brand check');
 
+promise_test(t => {
+
+  return Promise.all([
+    methodRejects(t, ReadableStream.prototype, 'cancel', null),
+    methodRejects(t, ReadableStream.prototype, 'cancel', undefined)
+  ]);
+
+}, 'ReadableStream brand checks do not throw on null or undefined');
+
+promise_test(t => {
+
+  return Promise.all([
+    methodRejects(t, ReadableStreamDefaultReader.prototype, 'read', undefined),
+    methodRejects(t, ReadableStreamDefaultReader.prototype, 'read', null)
+  ]);
+
+}, 'ReadableStreamDefaultReader brand checks do not throw on null or undefined');
+
+// There is no way to check ReadableStreamDefaultController brand checks for
```

Comment:
What I am concerned about is an implementation where `null` or `undefined` passes the brand check. I believe currently we have no test that will fail in that case.

In fact I wonder if instead of adding a new test, we want to modify every existing test to just supplement fakeXYZ() with another two assertions that pass null and undefined.